ocean_provider/routes/compute.py

```diff
@@ -240,9 +240,14 @@ def computeDelete():
         description: One or more of the required attributes are missing or invalid.
       401:
         description: Invalid asset data.
+      405:
+        description: Method rejected.
       503:
         description: Service Unavailable
     """
+    if request.method == "HEAD" or request.method == "OPTIONS":
```

Comment:
I remember I grouped these in a list called "safe methods" or something similar. You can use something like if request.method in safe_methods: ...

And then if we import these "safe methods" from another file you won't have to apply the change multiple times in the future, e.g. if a new restriction is made.

Reply:
There are no `safe_methods` or certain lists for it but I can do something like:

```
if request.method in ['HEAD', 'OPTIONS']
```

Reply:
yeah, I think you can create the list in some constants file. I would also make sure to use upper for request method.